pkgs/applications/science/logic/coq/default.nix

```diff
@@ -95,6 +95,7 @@ let
 self = stdenv.mkDerivation {
   pname = "coq";
   inherit (fetched) version src;
+  exact-version = args.version;
```

Comment:
I have a hard time following your code. What is the difference between `exact-version` and `version` (which is inherited).

Reply:
Right! Thanks for clarifying, I had forgotten about this.

Reply:
Well, me too actually, took me a few trials to get it right ;-).